src/Microsoft.AspNetCore/WebHost.cs

```diff
@@ -210,6 +210,12 @@ public static IWebHostBuilder CreateDefaultBuilder(string[] args)
 
                     services.AddTransient<IStartupFilter, HostFilteringStartupFilter>();
                 })
+#if NETCOREAPP2_2
```

Comment:
You don't actually need this conditional, Server.IIS targets netstandard2.0. If Server.IIS has specific TFM requirements they should be handled internally.